src/server/components/dashboard/router.ts

```diff
@@ -8,7 +8,7 @@ import {
   usersEditController,
   helpPageController,
   usersEditPostController,
-} from "./controllers";
+} from "./controllers/controllers";
```

Comment:
Would it makes sense to change the folder structure / filename in a future ticket?

Reply:
oopsie, this should be ./controllers so it reads the file ./controllers/index.ts. 

I don't particularly like this - although it was the best way of splitting up the listsController and the helpers and test it without making too much of a mess in the `/components/dashboard/listsItems` dir.

I also don't really like things being split up into MVC (models in the same place, views in the same place, controllers in the same place, especially not in the same file 😭). imo things should be split up by domain or resource. 

the controllers (based on how they're written with req/res/next, they should really be called "route handlers") should live closer to the router and/or the endpoint it relates to, like we've done in the past for new routes. (eg declarationGetController or `listsItems/item/update/controllers.confirm.ts`)

anyway, I'll amend the import.

but yes I think folder structure in general should be rethought